### PR TITLE
Feature/new UI theme - Login page, Create User page, Episode Search modal, Episode Search page, System Information page

### PR DESCRIPTION
**Pull request by jchue**
### Description

This is the sixth (and I believe the final!) PR for #158. It includes the Login and Create User (from invitation) pages, the Episode Search modal and mobile page (which I renamed from Podcast Search), and the System Information page (which I renamed from the Info page).

### Linked Issues

This concludes the remainder of #158!

### Additional context

There are some additional/smaller items, tech debt, and cleanup I want to do, but I can address them separately over time.

### CODE REVIEW

File: ui/src/language/json/de.json [hunk 1]
```diff
@@ -9,21 +9,21 @@
...
+  "cpu-usage": "CPU",
   "used-cpu": "Verwendete CPU",
   "free-cpu": "Freie CPU",
-  "memory-usage": "Speicherauslastung",
+  "memory-usage": "Speicher",
   "used-memory": "Verwendeter Speicher",
   "loading": "Laden",
   "free-memory": "Freier Speicher",
-  "disk-usage": "Festplattenauslastung",
+  "disk-usage": "Scheibe",
```
Comment (SamTV12345) [inline ui/src/language/json/de.json:23]: Should be Festplattenauslastung. Scheibe means Slice in English.

File: ui/src/language/json/de.json [hunk 2]
```diff
@@ -9,21 +9,21 @@
...
-  "cpu-info": "CPU Info",
+  "hardware": "Hardware",
   "cpu-brand": "CPU Marke",
   "cpu-cores": "CPU Kerne",
   "podcast-size": "Podcast Verzeichnis Größe",
-  "cpu-usage": "CPU-Auslastung",
+  "cpu-usage": "CPU",
   "used-cpu": "Verwendete CPU",
   "free-cpu": "Freie CPU",
-  "memory-usage": "Speicherauslastung",
```
Comment (SamTV12345) [inline ui/src/language/json/de.json:19]: Should be Speicherauslastung. Speicher means safe/storage

**Comment by SamTV12345** [comment]
There is somehow on every page a scrollbar which looks odd. Could you remove that. I don't know if you could make the sidebar a little bit less default. You could take a look at the https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar .

image

Other than that I really like the design and appreciate your continuos work throughout the last weeks.

File: ui/src/language/json/de.json (hunk 1 above)
Comment (jchue) [reply ui/src/language/json/de.json:23]: Oh, haha, thanks. Google Translate 🤣

**Comment by jchue** [comment]
> There is somehow on every page a scrollbar which looks odd. Could you remove that. I don't know if you could make the sidebar a little bit less default.

Hm, do you just mean the horizontal scrollbars on the homepage? Or are you also talking about the default vertical scrollbar on every page?

I don't usually like to mess with the default browser scrollbars. If it is only the horizontal scrollbars you are referring to, maybe it's better to just limit the number of items that are displayed and eliminating the scrollbars completely?

**Comment by SamTV12345** [comment]
> > There is somehow on every page a scrollbar which looks odd. Could you remove that. I don't know if you could make the sidebar a little bit less default.
> 
> Hm, do you just mean the horizontal scrollbars on the homepage? Or are you also talking about the default vertical scrollbar on every page?
> 
> I don't usually like to mess with the default browser scrollbars. If it is only the horizontal scrollbars you are referring to, maybe it's better to just limit the number of items that are displayed and eliminating the scrollbars completely?

The vertical bars are a bit weird as there is no content to scroll. Usually they are blended out on most pages if it is not required to scroll.

**Comment by jchue** [comment]
Hm, that is weird. I'll need to look into that. What OS and browser are you using?

**Comment by SamTV12345** [comment]
I'm using Windows 11, Chrome Version 114.0.5735.199 (OfficialBuild) (64-Bit). With zoom 100% I still see the x axis bar. 

image

The selected div is the error. When I set it to 80% the scrollbar shrinks and starts scrolling. So it is not the body but this div that causes the scrollbar to show up.

**Pull request merged**